pokemon/src/main/java/gamarket/PokemonCollection.java

```diff
@@ -117,23 +120,81 @@ public int getNumPokes()
 	
 	/**
 	 * A public function printCollection of type void.
-	 * The function uses a for loop to traverse the Pokemon array 
+	 * What it does: the function uses a for loop to traverse the Pokemon array 
 	 * and prints out the Pokemon Collection via its Pokemon class 
 	 * object status: identity, offensive, and defensive. 
 	 */
 	public void printCollection()
 	{
 		for(int i = 0; i < numPOKEs; i++)
 		{
-			System.out.println(pokeArray[i].toString());
+			System.out.println(pokeArray[i].toString("other"));
 		}
 	}
 	
+	/**
+	 * A public function printThemMoves of type void.
+	 * What it does: it prints out the Pokemon's name along with its four moves.
+	 * @return nothing. 
+	 */
+	public void printThemMoves()
+	{
+		for(int i = 0; i < numPOKEs; i++)
+		{
+			System.out.println(pokeArray[i].getIdentStats().getName() + "'s Move-Set:");
+			pokeArray[i].printPokemonMoves();
+			System.out.println(); 
+		}
+	}
+	
+	/**
+	 * A public function fillDefaultMoves of type void.
+	 * What it does: the function gives each Pokemon its default moves. 
+	 * @return nothing. 
+	 */
+	public void setDefaultMoves()
+	{	
+		// Flareon
+		Move m1 = mc.getMoveAtIndex(10);	// Fire Fang
+		Move m2 = mc.getMoveAtIndex(9);		// Flare blitz
+		Move m3 = mc.getMoveAtIndex(3); 	// Slam
+		Move m4 = mc.getMoveAtIndex(4);		// Quick Attack 
+		pokeArray[0].setDefaultMoves(m1, m2, m3, m4);
+		
+		// Gengar
+		m1 = mc.getMoveAtIndex(8);			// Phantom Force
+		m2 = mc.getMoveAtIndex(13);			// Physic
+		m3 = mc.getMoveAtIndex(7);  		// Drain Punch
+		m4 = mc.getMoveAtIndex(5);			// Shadow Ball
+		pokeArray[1].setDefaultMoves(m1, m2, m3, m4);
+		
+		// Pikachu
+		m1 = mc.getMoveAtIndex(0);			// Electric Ball	
+		m2 = mc.getMoveAtIndex(1);			// Thunder
+		m3 = mc.getMoveAtIndex(4);			// Quick Attack	
+		m4 = mc.getMoveAtIndex(3);			// Slam
+		pokeArray[2].setDefaultMoves(m1, m2, m3, m4);
+		
+		// Poliwrath 
+		m1 = mc.getMoveAtIndex(2);			// Water Pulse	
+		m2 = mc.getMoveAtIndex(12);			// Ice Punch	
+		m3 = mc.getMoveAtIndex(13);			// Physic
+		m4 = mc.getMoveAtIndex(14);			// Earthquake
+		pokeArray[3].setDefaultMoves(m1, m2, m3, m4);
+		
+		// Sandslash
+		m1 = mc.getMoveAtIndex(16);			// Slash	
+		m2 = mc.getMoveAtIndex(17);			// Drill Run
+		m3 = mc.getMoveAtIndex(18);			// Rock Slide
+		m4 = mc.getMoveAtIndex(12);			// Bite

```

Comment:
Use your mc.searchMove function to add by name instead of by index. If we add a new move this code won't work as intended.